### ISSUE TITLE
Transfer server

### ISSUE THREAD

**Issue by ashleyKing9** [purdue-epics-isd/TRACK#100]

**Comment by ashleyKing9** [comment]
Could attempt to create a "dummy" website on ECN server that automatically reroutes to Heroku. Then, make the index page of Heroku site only accessible if it gets a "key" passed from the dummy website???

**Comment by ashleyKing9** [comment]
This is what ECN support email told us:
NodeJS wants to run it's own web server, which is a configuration we
currently do not support. From what I read, NodeJS can be proxied
through an Apache server, which is how we manage the Engineering web
server, but we don't have an infrastucture for hosting standalone
NodeJS servers at this time. We can host, through Apache, php, python
and Rails web instances, but nothing else.
----------------------------------------------------------------
To summarize, we need to proxy NodeJS through Apache. However, all the directions for doing a proxy include editing the .conf file in the servers, but if you look at the ECN drive we have, the .conf file is restricted :( We can try to contact ECN about getting access, but that sounds like a lot of work... So maybe not

**Comment by ashleyKing9** [comment]
Something to also figure out: Does ISD use an Apache server? If not, then uploading to ECN might not even be helpful.
